include/snap.h

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef LABWC_SNAP_H
+#define LABWC_SNAP_H
+
+#include "config.h"
+#include <stdint.h>
+#include <wayland-util.h>
+#include <wlr/util/box.h>
```

Comment:
`<wayland-util.h>` and `<wlr/util/box.h>` can be removed.
To make it consistent and not rely on `view.h` including it, we can instead forward declare `struct wlr_box`.
We should however additionally include `common/border.h` here.

Reply:
Done (your [fixup])